Add Community Mentorship Summer 2020 Mentor Signups (#3862)
ppy/osu-wiki: news/2020-07-10-community-mentorship-program-summer-2020-signups-now-open.md

---
layout: post
title: Community Mentorship Program Summer 2020 Signups Now Open
date: 2020-07-10 08:00:00 +0000
---

> The 16th cycle of the Community Mentorship program is on its way! If you've ever found yourself wanting to expand your current knowledge and skills or help beginning mappers, this program will serve as a great opportunity for you!

Reviewer: ```suggestion
The 16th cycle of the Community Mentorship program is on its way! If you've ever found yourself wanting to expand your current knowledge and skills or help beginner mappers get started, this program will serve as a great opportunity for you!
```

![](/wiki/shared/news/banners/community-mentorship-program.jpg)

> The Community Mentorship program serves as place for upcoming mappers and modders to develop their skills underneath the guidance of more experienced mappers and modders in the community. Over the course of a cycle, you'll have the opportunity to learn from your mentor, participate in various mentorship events (like mapping contests), hang out with fellow mentees, and more!

Reviewer: ```suggestion
The Community Mentorship program serves as place for upcoming mappers and modders to develop their skills underneath the guidance of more experienced mappers and modders in the community. 

Over the course of a cycle, you'll have the opportunity to learn from your mentor, participate in various mentorship events (like mapping contests), hang out with fellow mentees, and more!
```

> If you're experienced in mapping and have a desire to take those who are new under your wing, then give joining us as Mentor a shot! In the [forum announcement and rule thread](https://osu.ppy.sh/community/forums/topics/1102406), you'll be able to apply right away. If you're only just getting into mapping or are eager to hone your skills, then signing up as Mentee might just be your way to do it!

Reviewer 2: ```suggestion
If you're experienced in mapping and have a desire to take those who are new under your wing, then give joining us as a Mentor a shot! In the [forum announcement and rule thread](https://osu.ppy.sh/community/forums/topics/1102406), you'll be able to apply right away. If you're only just getting into mapping or are eager to hone your skills, then signing up as Mentee might just be your way to do it!
```

---

For this following cycle, the timeline is as follows (all opening and closing dates start and end at midnight UTC+0):

- Mentor signups will open on July 13th and close on July 20th.
- Mentee signups will open on August 10th and close on August 16th.
- The 2020 Summer cycle will run from August 17th to November 20th.

Throughout this active cycle time, the organisation team will host various events on the Mentorship Discord server, engaging and offering events for mentees and mentors to participate in. These include feedback lobbies, mapping contests, and more! Some of our previous mapping puzzles can be found here at this [thread](https://osu.ppy.sh/community/forums/topics/565382), and previous discussions can be found [here](https://osu.ppy.sh/community/forums/topics/514122).

If any of this interests you or anyone you know, please go to our [forum announcement and rules thread](https://osu.ppy.sh/community/forums/topics/1102406) for more information.

We're looking forward to the mappers and modders of tomorrow!

> -yaspo

Reviewer: ```suggestion
—yaspo
```

---
layout: post
title: Community Mentorship Program Summer 2020 Signups Now Open
date: 2020-07-10 08:00:00 +0000
---

The 16th cycle of the Community Mentorship program is on its way! If you've ever found yourself wanting to expand your current knowledge and skills or help beginner mappers get started, this program will serve as a great opportunity for you!

![](/wiki/shared/news/banners/community-mentorship-program.jpg)

> The Community Mentorship program serves as place for upcoming mappers and modders to develop their skills underneath the guidance of more experienced mappers and modders in the community. 

Reviewer 3: ```suggestion
The Community Mentorship program serves as place for upcoming mappers and modders to develop their skills under the guidance of more experienced mappers and modders in the community. 
```
